base/genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_accessor :items, :name
+
+  def initialize(name)
+    @id = Random.rand(1..1000)
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+    item.add_genre(self)

```

Comment:
I can see that the add_genre method is applied to the item. Don't you think that it should also be set as an accessor on the Item class?

Reply:
Yes, I have created a `attr_accessor` for `genre`

Reply:
Ok thank you